### PR TITLE
Old NT Gun Army variants added to disks and vending machine + minor name fixes on autolathe disks

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
adds the army variants of the MK58 and Regulator to Disks and Billomat. as well as fixes some names for guns in autolathe disks and fixes the STS being called a battle rifle and not an assault rifle and not listing its caliber in the name

## Why It's Good For The Game
more variety (i forgot to add the mk58-m to the disk in the original PR) and some minor fixes

## Changelog
:cl:
tweak: MK58-m can now be found in the mk58 disk and billomat, Regulator M1000 can be found in its disk
spellcheck: fixed Excel Kalash, STS35, MK58c and Mandella names in autolathe printing
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
GUH

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 1]
```diff
@@ -330,6 +331,7 @@
 	license = 12
 	designs = list(
 		/datum/design/autolathe/gun/regulator = 3, // "NT SG \"Regulator 1000\""
+		/datum/design/autolathe/gun/regulator/army, = 3, //army version//
```
Comment (SirRichardFrancis) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:334]:
```suggestion
		/datum/design/autolathe/gun/regulator/army = 3, //army version//
```

**Review by SirRichardFrancis — CHANGES_REQUESTED** [review]
No testing?

image

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm (hunk 1 above)
Comment (Pink-Chink) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm]:
```suggestion
		/datum/design/autolathe/gun/regulator/army = 3, //army version//
```

Comment (ValoTheValo) [reply code/game/objects/items/weapons/design_disks/NeoTheology.dm]: for some reason VSC wont let me compile

Comment (ValoTheValo) [reply code/game/objects/items/weapons/design_disks/NeoTheology.dm]: okay so it was literally just a single comma. amazing

**Comment by Humonitarian** [comment]
STS is a battle rifle? It has a full-powered cartridge

**Review by Firefox13 — APPROVED** [review]
It's an salsault riffle

**Pull request merged**